src/react-components/home-root.js

```diff
@@ -184,6 +184,12 @@ class HomeRoot extends Component {
                   <a href="https://github.com/mozilla/hubs" rel="noreferrer noopener">
                     <FormattedMessage id="home.source_link" />
                   </a>
+                  <a href="/spoke" rel="noreferrer noopener">
+                    <FormattedMessage id="home.spoke_link" />
+                  </a>
+                  <a href="https://discord.gg/XzrGUY8" rel="noreferrer noopener">
+                    <FormattedMessage id="home.community_link" />
+                  </a>
```

Comment:
This link is covered by this seemingly unused `ident` div:

![image](https://user-images.githubusercontent.com/79419/46837535-93851500-cd6a-11e8-8c25-1ecaf3ae72e7.png)